storm-client/src/jvm/org/apache/storm/daemon/worker/Worker.java

```diff
@@ -246,7 +248,19 @@ public void start() throws Exception {
                             }
                         }
                     });
-              
+
+                workerState.checkForUpdatedBlobsTimer.scheduleRecurring(0,
```

Comment:
I wanted the period for this check to be it's own value and not tied to other timers